HomeWork1.py

```diff
@@ -3,4 +3,15 @@
 # *Пример:*
 
 # 123 -> 6 (1 + 2 + 3)
-# 100 -> 1 (1 + 0 + 0) |
\ No newline at end of file
+# 100 -> 1 (1 + 0 + 0) |
+
+import os # Делаем чистой консольку
+
+os.system('cls')
```

Comment:
Это будет работать не на всех платформах. Имейте ввиду. А еще это не будет работать через PyCharm

Reply:
а есть какой то универсальный способ?

Reply:
В большинстве случаев поможет это 
```suggestion
os.system('cls') if os.name == 'nt' else os.system('clear')
```
Это скрипт будет работать на windows, linux и mac. На остальных платформах надо смотреть команду для очистки терминала. Но на PyCharm по прежнем убудет работать не корректно, хотя это и не важно. Там все равно каждый запуск – в отдельном окне происходит.

Reply:
Благодарю!